### DISCUSSION TITLE
Missing Translation for captions of figure and table

### DISCUSSION THREAD

**Discussion by doc-slice** [brechtm/rinohtype#429] [Q&A]
In the Template Configuration, the language is set to German (language=de) and a number of labels (Chapter, Table of Contents etc.) are correctly translated. However, the labels of figures and tables as well as cross-references to them are still displayed as “figure” and “table”. How do I get a translation set here too?

**Comment by brechtm** [comment]
It looks like this is not yet implemented. I think the implementation would be very straightforward, but I can't make a promise as to when I would be able to get to that.

**Comment by brechtm** [comment]
This is now implemented in the development version. Please see [Getting Started](https://github.com/brechtm/rinohtype#getting-started) for how you can install it.

**Comment by doc-slice** [comment]
Many thanks for the quick adjustment. It works perfectly.
